Implement OrbitingCamera: MouseButtonToOrbit option

movitto opened the pull request:
Greetings! Thanks for the great Unity plugin.

This patch extends OrbitingCamera to support a flag wherein if set, it will require the left mouse button to be held to trigger orbiting functionality.

dynamicrealities commented:
Hello! and thanks for your contribution!
On line 25 you seem to have forgotten the finishing semi-colon. Fix that and I'll accept the pull request :)

movitto committed:
Implement OrbitingCamera: MouseButtonToOrbit option

If set, will require the left mouse button to be held to trigger
orbiting functionality.

movitto commented:
@dynamicrealities fixed!

dynamicrealities commented:
Thanks for the contribution :)
I might consider moving all of these scripts over to my new MIT repos as I'm just releasing scripts over there infrequently. Can find it here: https://gitlab.com/gamedev-public/unity
